### PR TITLE
Add Alter Solutions, Everis, Compare Europe Group

### PR DESCRIPTION
**Pull request by leplastic**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,11 +26,13 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 | [AddCode](https://addcode-io.breezy.hr) [:rocket:](https://addcode-io.breezy.hr/#positions) | Software engineering. | `Porto` |
 | [Affinity](https://affinity.pt) [:rocket:](https://affinity.pt/en/careers/) | Consultancy, Outsourcing. | `Lisboa` `Porto` |
 | [AGAP2](http://www.agap2-it.pt/Agap2IT) [:rocket:](http://www.agap2-it.pt/Agap2IT/pt/pages/carreira/) | Consultancy, Outsourcing. | `Lisboa` `Porto` |
+| [Alter Solutions](https://www.alter-solutions.com/pt-pt/) [:rocket:](https://www.linkedin.com/company/alter-solutions-portugal/) | Consultancy, outsourcing and nearshore projects. | `Lisboa` `Porto` |
```
Comment (marmelo) [inline README.md]: The Linkedin link is not a careers link, can you please check?

File: README.md [hunk 2]
```diff
@@ -106,6 +108,7 @@ Are you seeking for a job? Click :rocket: to check the company's careers page.
 
 | Company | Description | Locations |
 | :------ | :---------- | :-------- |
+| [Compare Europe Group](http://www.compareeuropegroup.com/) [:rocket:](http://www.compareeuropegroup.com/careers/) | Financial products comparision for several markets (ex: [ComparaJá](https://www.comparaja.pt)). | `Lisboa` |
```
Comment (marmelo) [inline README.md]: Is it possible to shorten the company name?
Also, is it possible to shorten the description to fit in a single row to improve readability?
Thanks!

File: README.md (hunk 1 above)
Comment (leplastic) [reply README.md]: Right now Linkedin seems to be the best way for candidates to contact Alter Solutions. It has a carrers page but it is not actively managed by the Portuguese team I think. I can link it instead but LinkedIN has more value for the purpose. Let me know what you prefer.

File: README.md (hunk 2 above)
Comment (leplastic) [reply README.md]: I shorten the company name and was able to fit the description in a single line

File: README.md (hunk 1 above)
Comment (marmelo) [reply README.md]: I have removed it because it is not a careers link. Thanks.

**Comment by marmelo** [comment]
Thanks @leplastic!

**Pull request merged**